src/lib/transaction_snark_scan_state/inputs.ml

```diff
@@ -85,7 +85,15 @@ module type S = sig
      and type account := Account.t
 
   module Sparse_ledger : sig
-    type t [@@deriving sexp, bin_io]
+    module Stable : sig
+      module V1 : sig
+        type t [@@deriving sexp, bin_io]
+      end
+
+      module Latest = V1
+    end
+
+    type t = Stable.V1.t
```

Comment:
It's better to use `Stable.Latest.t`, so that if/when `Latest` changes, we won't have to make a change here.

Reply:
what about this change?